jtec-core/src/main/java/edu/tum/sse/jtec/reporting/LCOVReport.java

```diff
@@ -0,0 +1,55 @@
+package edu.tum.sse.jtec.reporting;
+
+import java.util.List;
+
+public class LCOVReport {
+    private List<LCOVSection> lcovSections;
+
+    public LCOVReport(List<LCOVSection> lcovSections) {
+        this.lcovSections = lcovSections;
+    }
+
+    public List<LCOVSection> getLcovSections() {
+        return lcovSections;
+    }
+
+    public void setLcovSections(List<LCOVSection> lcovSections) {
+        this.lcovSections = lcovSections;
+    }
+
+    /**
+     * Get the LCOV section for the given source file.
+     * @param sourceFilePath The path to the source file
+     * @return The LCOV section if it exists, else null
+     */
+    public LCOVSection getLcovSection(String sourceFilePath) {
```

Comment:
We're trying to consistently use `Path`s from java.nio where possible. Maybe this makes sense here as well or don't we need the `Path` object anyways?

Reply:
Only the String is required here